Backend/app/database/DatabaseConnectionManager.py

```diff
@@ -41,15 +45,38 @@ def get_collection_connection(cls, collection_name: DatabaseCollection) -> Colle
     def init_database_connection(
         cls, environment: AppEnvironmentMode, connection_uri: str
     ) -> None:
-        """Initializes the database connection and loads its unique instance\
-            based on current environment value
+        """
+        Initializes the database connection.
 
         Args:
-            environment (AppEnvironmentMode): the current environment value
-            connection_uri (str): the database connection uri
+            environment (AppEnvironmentMode): The current environment mode.
+            connection_uri (str): The URI for connecting to the database.
+
+        Returns:
```

Comment:
There's no need to add docs for returning None. The extension njpwerner.autodocstring handles it like that and I think Pycharm too